src/main/resources/assets/computercraft/lua/rom/programs/http/webrun

```diff
@@ -1,25 +1,37 @@
---Made by Wilma456
-local args = {...}
+local Arg = {...}
 
-if not args[1] then
+if not Arg[1] then
   print("Usage: webrun <url> <args>")
-  return
+  return 1
 end
 
 if not http then
-  print("Please enable http in the config")
-  return
+print("Please enable http in the config")
+return 2
```

Comment:
Why are you arbitrarily returning `2`? There is a *nix specification for return codes. `0` for success, and `1` for generic error. `2` means misuse of shell builtins, which is not the problem here. Please read [this](http://tldp.org/LDP/abs/html/exitcodes.html)